### DISCUSSION TITLE
Wrapper for Rust

### DISCUSSION THREAD

**Discussion by ControlSoup** [CoolProp/CoolProp#2311] [General]
Hey all, I am a huge fan of this library and use it all the time for Python. 

I have recently switched over to Rust to take advantage of the type system for things like units. 

A wrapper for the rust application exists already: https://github.com/sgolle/coolprop-rs

But I was wondering if it would be possible to integrate this with the standard library and allow smoother installation/ package management.

**Comment by ibell** [comment]
I know nothing about packaging with rust, but if you wanted to take this on
and support it,  that would be great.
